Update tutorial.md (#104)
JuliaSmoothOptimizers/SolverBenchmark.jl: docs/src/tutorial.md

p = profile_solvers(stats, costs, costnames)
Plots.svg(p, "profile3")
```
### Example of benchmark running 

> Here is a useful tutorial on how to use the benchmark with specific solver.

Reviewer: ```suggestion
Here is a useful tutorial on how to use the benchmark with specific solver:
```

[Run a benchmark with OptimizationProblems](https://juliasmoothoptimizers.github.io/OptimizationProblems.jl/dev/benchmark/)

> In the tutorial they cover how to use the problems from OptimizationProblems to run a benchmark for unconstrained problems. 

Reviewer: ```suggestion
The tutorial covers how to use the problems from `OptimizationProblems` to run a benchmark for unconstrained optimization.
```
